tests/gunicorn/tox.ini

```diff
@@ -20,11 +20,11 @@ whitelist_externals =
 setenv =
   PYTHONPATH=../../src
   ENABLE_INSTRUMENTATION=False
-  HT_REPORTING_TRACE_REPORTER_TYPE=zipkin
+  HT_TRACES_EXPORTER=zipkin
   HT_REPORTING_ENDPOINT=http://localhost:9411/api/v2/spans
-  HT_LOG_LEVEL={env:HT_LOG_LEVEL:}
+  HT_LOG_LEVEL=DEBUG
```

Comment:
Curious about the reason for this change. This was introduced to let the user decide about the log level when running the tests.

Reply:
fixed